docs/spec/distribution/transactions.md

```diff
@@ -0,0 +1,396 @@
+triggered each with any change in individual delegations, such as an unbond,
+redelegation, or delegation of additional tokens to a specific validator.  
+
+Generally, each time a withdrawal is made by a recipient the adjustment term
+must be modified for each block with a change in distributors shares since the
+time of last withdrawal.  This is accomplished by iterating over all relevant
+`PowerChange`'s stored in distribution state.
+
+
+```golang
+type TxWithdrawDelegation struct {
+    delegatorAddr sdk.AccAddress
+    withdrawAddr  sdk.AccAddress // address to make the withdrawal to
+}
+
+func WithdrawDelegator(delegatorAddr, withdrawAddr sdk.AccAddress) 
+    entitlement = GetDelegatorEntitlement(delegatorAddr)
+    AddCoins(withdrawAddr, totalEntitlment.TruncateDecimal())
+
+func GetDelegatorEntitlement(delegatorAddr sdk.AccAddress) DecCoins
+    
+    // compile all the distribution scenarios
+    delegations = GetDelegations(delegatorAddr)
+    DelDistr = GetDelegationDistribution(delegation.DelegatorAddr,
+                                         delegation.ValidatorAddr)
+    pcs = GetPowerChanges(DelDistr.WithdrawalHeight)
+        
+    // update all adjustment factors for each delegation since last withdrawal
+    for pc = range pcs 
+        for delegation = range delegations 
+            DelDistr = GetDelegationDistribution(delegation.DelegatorAddr,
+                                                 delegation.ValidatorAddr)
+            pc.ProcessPowerChangeDelegation(delegation, DelDistr) 
+    
+    // collect all entitled fees
+    entitlement = 0
+    for delegation = range delegations 
+        global = GetGlobal() 
+        pool = GetPool() 
+        DelDistr = GetDelegationDistribution(delegation.DelegatorAddr,
+                        delegation.ValidatorAddr)
+        ValDistr = GetValidatorDistribution(delegation.ValidatorAddr)
+        validator = GetValidator(delegation.ValidatorAddr)
+
+        scenerio1 = NewDelegationFromGlobalPool(delegation, validator, 
+                        pool, global, ValDistr, DelDistr)
+        scenerio2 = NewDelegationFromProvisionPool(delegation, validator, 
+                        ValDistr, DelDistr)
+        entitlement += scenerio1.WithdrawalEntitlement()
+        entitlement += scenerio2.WithdrawalEntitlement()
+    
+    return entitlement
+
+func (pc PowerChange) ProcessPowerChangeDelegation(delegation sdk.Delegation, 
+                      DelDistr DelegationDistribution) 
+
+    // get the historical scenarios
+    scenario1 = pc.DelegationFromGlobalPool(delegation, DelDistr) 
+    scenario2 = pc.DelegationFromProvisionPool(delegation, DelDistr) 
+
+    // process the adjustment factors 
+    scenario1.UpdateAdjustmentForPowerChange(pc.Height) 
+    scenario2.UpdateAdjustmentForPowerChange(pc.Height) 
+```
+
+## TxWithdrawValidator
+
+When a validator wishes to withdraw their transaction fees it must send
```

Comment:
To clarify - this is *not* the same as withdrawing fees from the self-bond, which must be done separately, correct?